web_src/js/features/repo-issue.js

```diff
@@ -634,3 +635,42 @@ export function initRepoIssueBranchSelect() {
   };
   $('#branch-select > .item').on('click', changeBranchSelect);
 }
+
+export function initRepoIssueMarkdownTextarea() {
+  const hiddenTextarea = function (target) {
+    const comboMarkdownEditor = $(target).parent().find('.combo-markdown-editor');
+    const markdownTextEditor = comboMarkdownEditor.find('.markdown-text-editor');
+    const dropzone = $(target).parent().next().find('.dropzone');
+    // show combo markdown editor
+    comboMarkdownEditor.removeClass('gt-hidden');
+    markdownTextEditor.trigger('focus');
+    if (dropzone) {
+      dropzone.removeClass('gt-hidden');
+    }
+    // hidden textarea
+    $(target).addClass('gt-hidden');
+  };
+
+  const showTextarea = function (target) {
+    const comboMarkdownEditor = $(target).parent().find('.combo-markdown-editor');
+    const markdownTextEditor = comboMarkdownEditor.find('.markdown-text-editor');
+    const dropzone = $(target).parent().next().find('.dropzone');
+    // hidden combo markdown editor
+    $(comboMarkdownEditor).addClass('gt-hidden');
+    if (dropzone) {
+      $(dropzone).addClass('gt-hidden');
+    }
+    // show textarea
+    $(target).removeClass('gt-hidden');
+    // sync textarea content
+    $(target).val($(markdownTextEditor).val());
+  };
+
+  // default display textarea
+  $('.markdown-textarea .textarea').each((_, target) => showTextarea(target));
+
+  $('.markdown-textarea .textarea').on('focus', function() {
+    hiddenTextarea(this);
+    $('.markdown-textarea .textarea').not(this).each((_, target) => showTextarea(target));
+  });
```

Comment:
We need to treat the ComboMarkdownEditor as a sealed component, avoiding changing its internal state like this.

Otherwise the editor would break sooner or later. For example, the "switch editor" button also needs to work witht these states.

Reply:
I'm sorry I don't understand.
IMO only [here](https://github.com/go-gitea/gitea/pull/24400/files#diff-d39a63ccfd5f0fe703f74ce48a3624ba59a856317b0d236d7eae87dfcf6c1c20R646) changed ComboMarkdownEditor's internal state, as we need to focus on ComboMarkdownEditor's internal textarea when hide the fake textarea.